src/components/x-modal.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <BaseEventsModal :eventsToOpenModal="openEvents" class="x-modal" :animation="collapseFromTop">
```

Comment:
You don't need to add the `class="x-modal" here, as it is already added in the `x-component`. You can see this in the resulting html:

![image](https://user-images.githubusercontent.com/6247440/152151099-73505ab1-6b1f-415c-9932-eb846598a433.png)